tests/Stecman/Component/Symfony/Console/BashCompletion/CompletionHandlerTest.php

```diff
@@ -148,6 +173,10 @@ protected function createHandler($commandLine, $cursorIndex = null)
      */
     protected function getTerms($handlerOutput)
     {
+        if (null === $handlerOutput) {
```

Comment:
Maybe it's a bug in `runCompletion` method it in some cases (maybe when no matching completion handlers were found) it doesn't return anything which results in `null`.